react-app/src/styles/RecursiveComponent.module.css

```diff
@@ -0,0 +1,4 @@
+.recursive-component {
+    text-align: center;
+    font-size: 13px;
```

Comment:
I think that font size should be always responsive
![image](https://github.com/user-attachments/assets/761b24e8-487c-411c-a2f5-9b4e88400705)

Reply:
changed.